core/src/test/java/org/apache/spark/JavaJdbcRDDSuite.java

```diff
@@ -32,6 +33,7 @@
 import org.junit.Test;
 
 public class JavaJdbcRDDSuite implements Serializable {
+  private String dbName = "db_" + UUID.randomUUID().toString().replace('-', '_');
```

Comment:
I borrowed the logic. Thanks, @HyukjinKwon .

https://github.com/apache/spark/blob/2349175e1b81b0a61e1ed90c2d051c01cf78de9b/sql/core/src/test/scala/org/apache/spark/sql/test/SQLTestUtils.scala#L354